minitest-rails.gemspec

```diff
@@ -15,10 +15,10 @@ Gem::Specification.new do |gem|
                       ["README.md", "UPDATING.md", "CHANGELOG.md", "LICENSE"]
   gem.require_paths = ["lib"]
 
-  gem.required_ruby_version = ">= 2.7.0"
+  gem.required_ruby_version = ">= 3.1.0"
```

Comment:
also no

Reply:
I'll revert the changes as asked, but there is still a dependency on railties >= 7.2.0 which itself depends on Ruby >= 3.1 ([code](https://github.com/rails/rails/blob/v7.2.0/railties/railties.gemspec#L12)).